thentos-tests/src/Thentos/Test/Core.hs

```diff
@@ -200,22 +198,13 @@ loginAsGod actionState = do
     return (tok, credentials)
 
 
--- | Cloned from hspec-wai's 'request'.  (We don't want to use the
--- return type from there.)
-makeSRequest :: Method -> SBS -> [Header] -> LBS -> SRequest
-makeSRequest method path headers = SRequest req
-  where
-    req = setPath defaultRequest { requestMethod = method, requestHeaders = headers ++ defaultHeaders } path
-    defaultHeaders = [("Content-Type", "application/json")]
-
-
 -- * misc
 
 -- | Like 'Data.Aeson.decode' but allows all JSON values instead of just
 -- objects and arrays.
 --
--- Copied from https://github.com/haskell-servant/servant-client
--- (FIXME: also available from attoparsec these days.  replace!)
+-- Once we don't need snap any more we can upgrade to aeson >= 0.10 and use 'Aeson.decodeEither'
```

Comment:
'decodeEither' should read 'eitherDecode', I think (there is no decodeEither in aeson).